### PR TITLE
Fixed Recipes, Added Basalt Recipes

### PR DESCRIPTION
**Pull request by 0rc1nus**

### CODE REVIEW

**Review by Dariensg — COMMENTED** [review]
I believe Flenarn's PR takes care of all of this, but I'll have Swan double check to make sure I didn't miss anything. Flenarn's PR also refactors this, so I think we'd take his PR.

File: src/main/resources/data/infernalexp/recipes/basalt_button.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern":
+  [
+    "x  "
```
Comment (SwanX1) [inline src/main/resources/data/infernalexp/recipes/basalt_button.json:5]: no

File: src/main/resources/data/infernalexp/recipes/polished_basalt_pressure_plate.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern":
+  [
+    "xx "
```
Comment (SwanX1) [inline src/main/resources/data/infernalexp/recipes/polished_basalt_pressure_plate.json:5]:
```suggestion
    "xx"
```

**Review by SwanX1 — CHANGES_REQUESTED** [review]

**Comment by SwanX1** [comment]
Overriden by #63

**Pull request closed without merging**